frontend/components/ui/blocknote/blocks/quiz-multiple-choice-block.tsx

```
@@ -0,0 +1,177 @@
+          props: { options: updatedOptions as any },
+        });
+      };
+
+      const handleAddOption = () => {
+        const newOption = {
+          id: Date.now().toString(),
+          text: "",
+          isCorrect: false,
+        };
+        props.editor.updateBlock(props.block, {
+          props: { options: [...options, newOption] as any },
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)